cli/utils.ts

```diff
@@ -0,0 +1,40 @@
+import * as fs from "fs";
```

Comment:
These utils should probably go in the API, as we will use them from places where we only depend on the API and not the CLI.
Similarly above - the actual verification of the profiles should happen in the API package, not just the CLI.

Reply:
but in the API package we have already a profile instance, so the validation has already been passed.
or do you mean that we need to pass  only the path in the API package, and then read the file and perform the verification inside the API package?

Reply:
we can still pass in a profile instance rather than a filename - but it being correct depends on us being good Typescript developers! As a JS developer, I could of course just pass in any object I wanted, or just cast any old object in Typescript even.